### DISCUSSION TITLE
Welcome to ARCHEthic Node Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [archethic-foundation/archethic-node#197] [General]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

    Ask questions you’re wondering about.
    Share ideas.
    Engage with other community members.
    Welcome others and are open-minded. Remember that this is a community we
    build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ShervinR** [comment]
Hello Archethic-Foundation/Archethic-Node
Im  it bussines developement director , financial consoultant in digital
seg . Real question is what can i do for this beautifull project ?
Im will be happy to help grow archethic in any ways


Shervin.R

**Reply by ghost** [reply]
Hello. Thanks for your message.
Sorry for the delay, we were quite busy to implement and stabilize the network.
We will be glad to be in touch
